lib/workers/pr/index.ts

```diff
@@ -309,17 +317,21 @@ export async function ensurePr(
 
   config.hasReleaseNotes = config.upgrades.some((upg) => upg.hasReleaseNotes);
 
-  const releaseNoteRepos: string[] = [];
+  const releaseNotesSources: string[] = [];
   for (const upgrade of config.upgrades) {
-    if (upgrade.hasReleaseNotes) {
-      if (releaseNoteRepos.includes(upgrade.sourceUrl)) {
+    const notesSourceUrl =
+      upgrade.releases?.[0]?.releaseNotes?.notesSourceUrl || upgrade.sourceUrl;
+
+    // istanbul ignore if
```

Comment:
Can you make a test instead?

Reply:
in progress

Reply:
done. i updated an existing test `should create group PR`